File: plotting_ind_stats.py
library(ggplot2)


ind_stats_file = "C:/Users/IPSEG/Desktop/Waples/chum_populations/ind_seq_stats.tsv"
ind_stats <- read.table(file = ind_stats_file, sep = "\t", header = TRUE)


ggplot(data = ind_stats, aes(x = pop, y= log10(nreads))) + 
  geom_boxplot() 
  
  fill = pop, color = pop   
  
ggplot(data = ind_stats, aes(x = pop, y= lenreads, fill = pop, color = pop )) + 
  geom_boxplot()
  
  facet_grid(popsize_factor ~ starting_loci_factor, labeller = label_both, scales = "free_y") + 
  labs(title = "Effecive Size estimated from all loci", y= 'Ne', x= "Number of chromosomes")

File: plotting_ind_stats.R
library(ggplot2)


ind_stats_file = "C:/Users/IPSEG/Desktop/Waples/chum_populations/ind_seq_stats.tsv"
ind_stats <- read.table(file = ind_stats_file, sep = "\t", header = TRUE)

ggplot(data = ind_stats, aes(x= log10(nreads))) + 
  geom_histogram() 

ggplot(data = ind_stats, aes(x = pop, y= log10(nreads), fill = factor(pop) )) + 
  geom_violin(scale = 'width', trim = TRUE, adjust = 1) + coord_flip() 
 
ggplot(data = ind_stats, aes(x = pop, y= log10(nreads), fill = factor(pop) )) + 
  geom_violin(scale = 'width', trim = TRUE, adjust = 1) + coord_flip()

above = table(ind_stats[ind_stats$nreads > 750000,]$pop)
below = table(ind_stats[ind_stats$nreads < 750000,]$pop)

plot(above/(above+below))

